lib/jackalope/tortoise_client.ex

```diff
@@ -201,6 +209,58 @@ defmodule Jackalope.TortoiseClient do
     {:noreply, state}
   end
 
+  def handle_info(:capture_telemetry, state) do
+    Process.send_after(self(), :capture_telemetry, @telemetry_delay)
+    {:noreply, capture_telemetry(state)}
+  end
+
+  defp capture_telemetry(state) do
+    client_id = state.client_id
+    registered_name = Tortoise311.Registry.via_name(Tortoise311.Connection, client_id)
+
+    case Tortoise311.Registry.meta(registered_name) do
+      {:ok, {_transport, socket}} ->
+        case ssl_socket_stats(socket) do
+          {:ok, stats} ->
+            Logger.debug("[Jackalope] inet stats #{inspect(stats)} for socket #{inspect(socket)}")
+            {received, sent} = new_stats(stats, state.socket_stats, socket)
+            Logger.debug("[Jackalope] Received #{received} bytes and sent #{sent} bytes")
+
+            :telemetry.execute([:jackalope, :transmission], %{
+              received: received,
+              sent: sent,
+              sent_and_received: received + sent
```

Comment:
```suggestion
              recv_oct: received,
              send_oct: sent
```

These aren't the best names, but I do like them since I know that they're reporting byte counts rather than packet counts. The `send_and_received` metric seems like it should be calculated by the code showing the reports rather than stored here.